### PR TITLE
Laravel 9.8 support

### PR DESCRIPTION
**Pull request by Null78**
updated the packages and fixed some comments
to support latest version of Laravel 9.8

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -16,13 +16,13 @@
         }
     ],
     "require": {
-        "php": "^7.1 || ^7.3 || ^8.0",
-        "aws/aws-sdk-php": "^3.145",
-        "illuminate/support": "5.7.* || 5.8.* ||^6.0 || ^7.0 || ^8.0"
+        "php": "^8.0",
+        "aws/aws-sdk-php": "^3.219",
+        "illuminate/support": "^9.8"
```
Comment (swilla) [inline composer.json:21]: Can you add back the previous version support? 

```suggestion
        "php": "^7.4 || ^8.0",
        "aws/aws-sdk-php": "^3.219",
        "illuminate/support": "^6.0 || ^7.0 || ^8.0 || ^9.8"
```

**Review by swilla — CHANGES_REQUESTED** [review]
Thank you! I'd rather not break previous version support. Can you update the dependencies?

**Comment by swilla** [comment]
Also, need to update the workflow matrix to run tests on L9 too: 
https://github.com/TappNetwork/laravel-aws-secrets-manager/blob/master/.github/workflows/run-tests-l8.yml

**Comment by Null78** [comment]
> Also, need to update the workflow matrix to run tests on L9 too: https://github.com/TappNetwork/laravel-aws-secrets-manager/blob/master/.github/workflows/run-tests-l8.yml

can you please check it ?

File: composer.json [hunk 2]
```diff
@@ -16,13 +16,13 @@
...
-        "aws/aws-sdk-php": "^3.145",
-        "illuminate/support": "5.7.* || 5.8.* ||^6.0 || ^7.0 || ^8.0"
+        "php": "^7.4 || ^8.0",
+        "aws/aws-sdk-php": "^3.219",
+        "illuminate/support": "^6.0 || ^7.0 || ^8.0 || ^9.8"
     },
     "require-dev": {
-        "orchestra/testbench": "^3.8 || ^5.0 || ^6.0",
-        "phpunit/phpunit": "^7.0 || ^8.4 || ^9.3.3"
+        "orchestra/testbench": "^7.4",
```
Comment (swilla) [inline composer.json:24]: I think you need multiple versions here. I approved the actions to run and you can see which ones are failing now.

**Review by swilla — CHANGES_REQUESTED** [review]

File: .github/workflows/run-tests-l9.yml [hunk 3]
```diff
@@ -10,11 +10,13 @@ jobs:
...
       matrix:
         php: [8.0, 7.4, 7.3]
-        laravel: [8.*]
+        laravel: [8.*, 9.*]
         dependency-version: [prefer-lowest, prefer-stable]
         include:
           - laravel: 8.*
             testbench: 6.*
+          - laravel: 9.*
+            testbench: 7.*
```
Comment (swilla) [inline .github/workflows/run-tests-l9.yml:20]: Can you add exclude: 
```
exclude:
    - laravel: 9.*
      php: 7.* 
```

**Review by swilla — CHANGES_REQUESTED** [review]

**Review by swilla — APPROVED** [review]

**Comment by Null78** [comment]
ok I actually removed some commits because I was testing
but you merged anyway

thanks

**Comment by swilla** [comment]
All the tests were passing for me. Let me know if I missed something.

**Pull request merged**